### DISCUSSION TITLE
My Blue theme

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#2853] [Show and tell]
I shared a blue theme with the following features:
- More compact, line spacing, paragraph spacing, etc.
- Indent the first line and align both ends.
- The directory tree mouse displays the background color when hovering to highlight the selected folder.
- Tree style.
- Title style of H1, H2, H3 and H4 .
- More obvious scroll bar.
- Expand the background image of the tree.
- Other changes.
- **Added dark mode, optional adaptive inversion of white picture color.**
Available [here](https://github.com/SiriusXT/trilium-theme-blue). Hope you like it! 

![screenshot](https://user-images.githubusercontent.com/37627919/168465011-74a1186b-fa52-422f-b616-dad8e23c5f0a.png)
![screenshot2](https://user-images.githubusercontent.com/37627919/229730830-121b66d0-1f93-4396-bc7d-cd60db13b097.png)

**Comment by Nriver** [comment]
added to https://github.com/Nriver/awesome-trilium
